Sofa/Component/Constraint/Lagrangian/Solver/src/sofa/component/constraint/lagrangian/solver/visitors/MechanicalGetConstraintResolutionVisitor.h

```diff
@@ -48,6 +48,7 @@ class SOFA_COMPONENT_CONSTRAINT_LAGRANGIAN_SOLVER_API MechanicalGetConstraintRes
 private:
     /// Constraint parameters
     const sofa::core::ConstraintParams *cparams;
+    const DeprecatedAndRemoved _cparams{};
```

Comment:
Since the member is private, I would not bother with deprecation

Reply:
Sir yes sir 👮